content/02.main-text.md

```diff
@@ -69,3 +69,31 @@ The randomization procedure was shared with the authors in advance (pre-register
+`TODO: need help deciding what related topics to include here and which references to use, these are arbitrary examples`
+`TODO: more ideas in doi:10.7287/peerj.preprints.2711v2`
+Several of these open science efforts are GitHub-based like our collaborative writing process.
+The Journal of Open Source Software [@arxiv:1707.02264] and ReScience [@arxiv:1707.04393] journals rely on GitHub for peer review and hosting.
+`TODO: describe Manubot related work here?` [@doi:10.7717/peerj-cs.112; @url:https://github.com/ewanmellor/gh-publisher]
+
+There are potential limitations of our GitHub-based approach.
+Because our review manuscript pertained to a computational topic, most of the authors had computational backgrounds, including previous experience with version control workflows and GitHub.
+In other disciplines, collaborative writing via GitHub and Manubot could present a steeper barrier to entry and deter participants.
+In addition, git carefully tracks all revisions to the manuscript text but not the surrounding conversations that take place through GitHub issues and pull requests.
+These discussions must be archived to ensure that important decisions about the manuscript are preserved and authors receive credit for intellectual contributions that are not directly reflected in the manuscript's text.
+GitHub supports programmatic access to issues, pull requests, and reviews so tracking these conversations is feasible in the future.
+`TODO: did not discuss conflict management and diversity, which are in the outline`
+
+Open writing presents new opportunities for scholarly communication.
+`TODO: reference "paper of the future"? arXiv:1601.02927 doi:10.22541/au.149693987.70506124 doi:10.22541/au.148769949.92783646 http://blogs.nature.com/naturejobs/2017/06/01/techblog-c-titus-brown-predicting-the-paper-of-the-future`
+Though it is still valuable to have versioned drafts of a review manuscript with digital identifiers, journal publication may not be the terminal endpoint for collaborative manuscripts.
+After releasing the first version of our collaborative review [@doi:10.1101/142760], six new authors have contributed text and existing authors continue to discuss new literature, creating a living document [@url:https://github.com/greenelab/deep-review/].
+`TODO: update new author count before submitting`
+The Manubot system can also facilitate open research [@doi:10.7287/peerj.preprints.3100v1] in addition to review articles.
+`TODO: get permission and add https://slochower.github.io/motor-like-properties-response/ https://slochower.github.io/nonequilibrium-barrier/ https://zietzm.github.io/Vagelos2017/`
+
+Our process represents an early step toward massively collaborative review writing, and there are certainly aspects that can be improved.
```

Comment:
I would like the word `open` somewhere in the "massively collaborative review" bit, because I think that's an important element of what we set out to do. I'm not sure "writing" is necessary if you go with `reviews`, but that i don't feel strongly on.